WeakAuras/BuffTrigger2.lua

```diff
@@ -303,13 +329,16 @@ local function UpdateMatchData(time, matchDataChanged, unit, index, filter, name
       lastChanged = time,
       filter = filter,
       index = index,
+      points = points,
```

Comment:
So I looked at all the other entries in the structure and points is the most interesting to me.

So points is as far as I can see the replacement for the additional return values that some UnitAura calls had. For e.g. Power Shield, the absorb value is returned as the first "point" value. In general it appears that for an aura there's one "point" per effect, e.g. https://www.wowhead.com/de/spell=397734 has 8 effects listed and thus point will contain 8 values. Or Avatar https://www.wowhead.com/de/spell=107574 there will be 7 values.

This isn't always the same information as is in the tooltip, sometimes tooltip values appear to come from somewhere else.

Now, since it's table, it's actually not directly useable as a text replacement and there are several ways we handle that
1) Do nothing at all, we ignore points
2) Add points to the state table, %c can retrieve them and display them but that's it. (And maybe some day in the future add text replacements that can deal with a table, e.g. something like %points.1 (?))
3) Add e.g. point1, point2, point3 as needed that refers to the individual members

I think 2) is actually reasonable for now. I don't have a good reason to believe that they would be that useful, but this gives enough support that people can try using the information.

Reply:
agreed for 2)